### ISSUE TITLE
Deploy Automat-CAM-KP to RENCI and ITRB

### ISSUE THREAD

**Issue by gaurav** [ExposuresProvider/cam-pipeline#90]
- [x] Prepare a CAM-KP Automat instance in RENCI-exp (https://automat-dev.apps.renci.org/#/cam-kp)
- [x] #91
- [x] Deploy to CAM-KP RENCI-dev (https://automat.renci.org/#/cam-kp)
- [x] Update our SmartAPI registration at http://smart-api.info/registry?q=642b1dbcd0943f3a119d714cf7535fba  if possible -- we'll probably need to delete it and create a new registration for the Automat endpoint.
- [x] Deploy to Automat ITRB CI (https://automat.ci.transltr.io/)
  - Probably best to do this via #devops-sri-ranking along with the rest of Automat 
- [x] Update our SmartAPI registration at http://smart-api.info/registry?q=c409da43f5177ae52392e2b346dbc585 if possible -- we'll probably need to delete it and create a new registration for the Automat endpoint.
- [x] Put in an ITRB request to deploy to ITRB Test and ITRB Prod (deadline: Mondays at noon ET)
  - Probably best to do this via #devops-sri-ranking along with the rest of Automat 
- [x] Delete our existing SmartAPI registrations at http://smart-api.info/registry?q=4803457bdb4bfeeb63a88244830ece2e and http://smart-api.info/registry?q=3d41210995d6cf1c97f355e678b9b09f and replace them with new SmartAPI registrations for the Automat-CAM-KP.

**Comment by EvanDietzMorris** [comment]
Without some redirections implemented the URLs will have to change, but the platers do generate their own openapi spec so it should be relatively easy to register for smartapi. @YaphetKG knows way more about this step than me though.

**Comment by EvanDietzMorris** [comment]
There is some complication here surrounding ITRB deployments.. currently all of the plater helm charts live here https://github.com/helxplatform/translator-devops and we make requests for pushing to ITRB test and prod through a different ITRB slack channel, typically many platers at a time (CI deployments happen automatically with jenkins when commits are merged to a certain branch in that devops repo). It shouldn't be a problem to push cam kp on it's own but it might be confusing to cross the request channels (at least from ITRBs perspective). Also - we don't always deploy a dev/prod version of every plater on Sterling, it looks like we call what you're calling renci-prod dev, but we could for you.

**Comment by gaurav** [comment]
Hi everybody! I've been playing around with Automat-CAM-KP in #91. The good news is that two of the test queries work fine, and two that produce an internal error both use deprecated Biolink 2 predicates. A few queries return . Queries with Biolink 3 predicates (with qualifiers) return zero results, which is not surprising. Two queries return results in CAM-KP-API but not in Automat-CAM-KP, which might be caused by modeling issues or maybe just missing data. Although CAM-KP-API is still better overall than Automat-CAM-KP, I think all the extra features we get -- including TRAPI 1.4 support eventually! -- means that we should go ahead and roll this out at least to ITRB CI while we try to get qualified predicates to work and while we try to figure out why those two queries don't work. Does that sound right?

I also tried some of the SRI testing data at random, and I was always able to retrieve the testing data using the query endpoint. And I've confirmed that `https://automat-dev.apps.renci.org/cam-kp/1.3` could be used as an OTHER endpoint at https://arax.ncats.io/. So this is looking pretty good!

@EvanDietzMorris When are y'all planning to deploy to Automat on RENCI next? Could we include CAM-KP in that deployment?
